### PR TITLE
Java tests refactoring

### PR DESCRIPTION
**Pull request by saberduck**
- remove dependency between uast-generator-java and commons-rule-engine
 - commit generated uast files for java sources

### CODE REVIEW

File: uast-generator-java/src/main/java/org/sonar/uast/generator/java/Generator.java [hunk 1]
```diff
@@ -46,6 +52,24 @@ public Generator(String source) {
     cut.accept(new PostprocessVisitor());
   }
 
+  private static String readFile(Path path) {
```
Comment (ghost) [inline uast-generator-java/src/main/java/org/sonar/uast/generator/java/Generator.java:55]: Can't see any unit test for these two methods.

File: uast-generator-java/build.gradle [hunk 2]
```diff
@@ -28,10 +28,19 @@ dependencies {
     implementation("org.sonarsource.java:java-frontend:${javaFrontendVersion}") {
         exclude group: 'junit', module: 'junit'
     }
-    implementation project(":common-rule-engine")
 
     testCompile "org.junit.jupiter:junit-jupiter-api:${junitJupiterVersion}"
     testRuntime "org.junit.jupiter:junit-jupiter-engine:${junitJupiterVersion}"
     // to run tests on eclipse...
-    testCompile group: 'org.junit.platform', name: 'junit-platform-launcher', version: "1.1.0"
+    testCompile "org.junit.platform:junit-platform-launcher:1.1.0"
```
Comment (ghost) [inline uast-generator-java/build.gradle:35]: :+1:

**Comment by ghost** [comment]
Except from the two methods you added to generate file which may deserve proper unit testing, LGTM.

File: uast-generator-java/src/main/java/org/sonar/uast/generator/java/Generator.java (hunk 1 above)
Comment (saberduck) [reply uast-generator-java/src/main/java/org/sonar/uast/generator/java/Generator.java:55]: That's normal, I didn't add any ;-)

**Pull request merged**